pkg/sbproxy/reconcile/reconcile_visibilities.go

```diff
@@ -45,33 +44,52 @@ func (r *ReconciliationTask) processVisibilities() {
 		return
 	}
 
-	plans, err := r.getSMPlans()
+	smBrokers, ok := r.stat(smBrokersStats).([]platform.ServiceBroker)
+	if !ok {
+		logger.Error("Could not get SM brokers from stats")
```

Comment:
Can we use `Service Manager` here (and all other occurrences) instead of `SM` due to consistency. `SM` seems a little informal as well